velox/dwio/parquet/tests/reader/E2EFilterTest.cpp

```diff
@@ -91,6 +91,20 @@ TEST_F(E2EFilterTest, writerMagic) {
   EXPECT_EQ("PAR1", std::string(data + size - 4, 4));
 }
 
+TEST_F(E2EFilterTest, booleanDirect) {
+  writerProperties_ = ::parquet::WriterProperties::Builder()
+                          .disable_dictionary()
```

Comment:
So if we enable dictionary, it will actually start generating dictionary-encoded bits?  If that is the case, we need to support it